tests/data/csvs/conservation_areas_invalid.csv

```diff
@@ -0,0 +1,3 @@
+geometry,id,address,area(ha),link,Layer

```

Comment:
for some of these you could look at creating this in the test code using fixtures. Although I don't think it's a major need.